content/20.treatments.md

```diff
@@ -211,19 +211,18 @@ Brief background on the therapeutic.
 ##### Drugs targeting a recombinant IL-1 receptor
 ###### Anakinra
 **Anticipated Mechanism.**
-Anakinra is a recombinant of IL-1 receptor antagonist (IL-1Ra) that blocks binding of IL-1α, or IL-1β to IL-1R1. Its main task is to reduce inflammation caused by IL-1-mediation, which finds a broad spectrum of applications, including acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157]. The application of Anakinra features a very safety profile, and has a very low rate of side effects and prolonged duration. 
+Anakinra is a recombinant of IL-1 receptor antagonist (IL-1Ra) that blocks binding of IL-1α, or IL-1β to IL-1R1. 
+Its main task is to reduce inflammation caused by IL-1-mediation, which finds a broad spectrum of applications, including acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157]. 
+The application of Anakinra features a very safety profile, and has a very low rate of side effects and prolonged duration. 
 
 **Current Evidence.**
 
-In two retrospective cohort studies, first at the San Raffaele Hospital in Milan, Italy with 29 patients treated with high dosages of anakinra and 16 controls [@doi:10.1016/S2665-9913(20)30127-2], and second at Groupe Hospitalier Paris Saint-Joseph Paris, France, with 96 participants in France, among which 52 were treated with the drug and 44 were administered standard protocols, anakinra has proven to 
-progressively improve respiratory function, dicrease likelihood of application of invasive mechanical ventilation, and significantly reduce mortality [@doi:10.1016/S2665-9913(20)30164-8].
+In two retrospective cohort studies, first at the San Raffaele Hospital in Milan, Italy with 29 patients treated with high dosages of anakinra and 16 controls [@doi:10.1016/S2665-9913(20)30127-2], and second at Groupe Hospitalier Paris Saint-Joseph Paris, France, with 96 participants in France, among which 52 were treated with the drug and 44 were administered standard protocols, anakinra has proven to progressively improve respiratory function, dicrease likelihood of application of invasive mechanical ventilation, and significantly reduce mortality [@doi:10.1016/S2665-9913(20)30164-8].
```

Comment:
In the recent randomized clinical trial, anakinra was found to be ineffective for COVID-19 and mild-to-moderate pneumonia patients treated intravenously with anakinra, although the studies used a different treatment protocol than the previous studies (half of the dosage, which was admitted intravenously). The authors didn't exclude anakinra effectiveness for severe COVID-19 cases and commented a decreased dosage as the  limitation of their study [@10.1016/S2213-2600(20)30556-7].